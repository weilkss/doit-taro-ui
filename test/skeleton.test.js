import React from 'react'
import { renderToString } from 'nerv-server'
import { Skeleton } from '../lib'

describe('Skeleton', () => {
  it('render Skeleton -- props type(column)', () => {
    const component = renderToString(
      <Skeleton type='column'>
        <p>Skeleton</p>
      </Skeleton>
    )
    expect(component).toMatchSnapshot()
  })
})
describe('Skeleton', () => {
  it('render Skeleton -- props avatar(true)', () => {
    const component = renderToString(
      <Skeleton avatar={true}>
        <p>Skeleton</p>
      </Skeleton>
    )
    expect(component).toMatchSnapshot()
  })
})
describe('Skeleton', () => {
  it('render Skeleton -- props contentAlignStyle(left)', () => {
    const component = renderToString(
      <Skeleton contentAlignStyle='left'>
        <p>Skeleton</p>
      </Skeleton>
    )
    expect(component).toMatchSnapshot()
  })
})
