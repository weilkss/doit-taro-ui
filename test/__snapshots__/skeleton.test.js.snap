// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton render Skeleton -- props contentAlignStyle(left) 1`] = `undefined`;

exports[`Skeleton render Skeleton -- props type(column) 1`] = `undefined`;
